index.html

```diff
@@ -2214,17 +2214,11 @@ <h2>The NDEFReader object</h2>
 
   <section><h3>Handling visibility change</h3>
     <p>
-      When the user agent <a href="https://html.spec.whatwg.org/#update-the-visibility-state">updates the visibility state</a>
-      of the <a>current settings object</a>'s <a>relevant global object</a>'s
-      <a>associated <code>Document</code></a> it must run these steps:
+      The <a href="https://html.spec.whatwg.org/#page-visibility-change-steps">page visibility change steps</a> for this specification, given |visibilityState| and |document|, are:
```

Comment:
Not sure if it's good practice to link the `visibilityState` and `document` variables to `Document/visibilityState` and `Document` respectively.

Reply:
I don't think so. You could mention their types, e.g. `given a string |visibilityState| and a {{Document}} |document|`? But the variables themselves should not be links.